npm/private/npm_translate_lock_generate.bzl

```diff
@@ -82,6 +82,10 @@ _BZL_LIBRARY_TMPL = \
 
 _PACKAGE_JSON_BZL_FILENAME = "package_json.bzl"
 
+################################################################################
+def _to_apparent_repo_name(canonical_name):
+    return canonical_name[canonical_name.rfind("~") + 1:]
```

Comment:
The "apparent" repo name is the nomenclature used in https://docs.google.com/document/d/1N81qfCa8oskCk5LqTW-LNthy6EBrDot7bdUsjz6JFC4/edit#heading=h.kzp6mqja2p3t.

Reply:
Could consider extracting this into bazel-lib.